src/screens/home-feed.tsx

```diff
@@ -65,6 +66,12 @@ export const HomeFeedScreen = () => {
 
   useEffect(() => {}, [wallets]);
 
+  const filterTransactions = useCallback(() => {
+    return transactionsList.filter(
+      ({providerId}) => user.providerId === providerId,
+    );
+  }, [transactionsList, user.providerId])();
+
```

Comment:
Not solve problem, if we have operations for day X and have no operations in other network for these day, we saw header for day X